### ISSUE TITLE
Trigger malware detection upon download

### ISSUE THREAD

**Issue by ZZZXXX0110** [blackholeearth/Win10_BrightnessSlider#13]
Identified by Windows Defender (threat definition version: 1.263.278.0, created on 10:26:35, March 7, 2018) as Trojan:Win32/Spursint.F!cl, upon download.

Windows 10 Version: Insider Build 17112

**Comment by chippie** [comment]
I get this also, see the closed Issues. The author is adamant there is no virus and I can download on my work laptop without issue however Defender will just not let it through or even live on any of my drives with this build of Windows. Guess there is an issue or we just have to wait for M$ to update Defender so it does not block it....

**Comment by blackholeearth** [comment]
**short:**
1)  a) disable antivirus; then download. 
   b) place exe to its final location (dont move again. if moved, whitelist and runt at startup may/will fail. ).  
   c) white list the file in antivirus.

2) wait for win defender update. (try update)
3) use other antivirus.
-------------------
**long:**

i subbmitted the file to microsoft winDefender:
 https://www.microsoft.com/en-us/wdsi/filesubmission 
it said that its not malware. approved by human too. if they put signature or *whatever they do* in windefender update . you wont  be blocked by it.  

scan file at virus total . 33 antivrus  all clean.  all of 33 cant be wrong!!.

----------------------
duplicate of  #12

**Comment by amartens181** [comment]
@ZZZXXX0110 Sometimes Windows Defender Marks files with that for no reason... ~4/1000 have gotten that from my program that isn't a virus. They were all in countries other than the US. That code is almost always a false positive.

**Comment by ZZZXXX0110** [comment]
And the funny thing is, it seems to only get marked false positive on Build 17112, on my other PC running release version 1709 (build 16299) it got downloaded just fine. This could indeed be a problem on Microsoft's side.

**Comment by chippie** [comment]
It definitely is. The problem is that it will not even let the the .exe sit anywhere on the drive. Cannot exclude it either. Will just have to wait for the build to update I guess.

**Comment by blackholeearth** [comment]
@chippie   _The problem is that it will not even let the the .exe sit anywhere on the drive._

 **while defender is active. you wont be able to do anything.**   
First, disable defender . Then you can download it .  add exclusion. then   activate defender.

 **step by step**:

1
windows icon at taskbar>settings> security update> windows defender
**disable** realtime protection.
**disable**  cloudbase submission

2
open firefox/chrome  browser> download file  > move file to convenient location.

3
tutorial for exclusion  : https://www.youtube.com/watch?v=lLBCOnt8bEA

add exclusion FILE: 
after you add file it will show  :   `C:\User\download\win_brislider.exe     [Remove]`

**IMPORTANT: if you move/relocate the winbrislider .exe** ,   filepath  exclusion  wont know the new location.  you will get virus alert.  so  dont move file after you do whitelisting exe. 

if you want to move the file, before you move file
     go to step 1 (at step 2 you are relocation your file)


 




have you even tried what i said.
